detection_rules/etc/non-ecs-schema.json

```diff
@@ -56,5 +56,9 @@
   },
   "logs-windows.*": {
     "powershell.file.script_block_text": "text"
+  },
+  "logs-kubernetes.*": {
+    "kubernetes.audit.objectRef.resource": "keyword",
+    "kubernetes.audit.objectRef.subresource": "keyword"
```

Comment:
You may not have many options but you should be aware that we will be reviewing the fields used in non-ecs-schema.

https://github.com/elastic/detection-rules/issues/1776